packages/functional-tests/.eslintrc.js

```diff
@@ -4,14 +4,19 @@ module.exports = {
     browser: true,
   },
   extends: [
-    // 'eslint:recommended',
-    // 'plugin:playwright/recommended',
+    'eslint:recommended',
+    'plugin:playwright/recommended'
   ],
+  globals: {
+    globalThis: false, // not writeable
+  },
   rules: {
     '@typescript-eslint/no-unused-vars': 'warn',
     '@typescript-eslint/ban-ts-comment': 'off',
     '@typescript-eslint/ban-types': 'warn',
-    // 'playwright/missing-playwright-await': 'off',
-    'no-console': ['error', { allow: ['log'] }], // console errors cause side effects in CircleCI
+    'playwright/missing-playwright-await': 'off',
+    'playwright/no-skipped-test': ['error', { allowConditional: true }],
+    'no-console': ['error', { allow: ['log'] }], // console errors cause side effects in CircleCI (FXA-8773)
+    'no-unused-vars': 'off',
```

Comment:
Turning this off since we have `@typescript-eslint/no-unused-vars` and they were throwing different results, and that seems to be the recommendation, per https://typescript-eslint.io/rules/no-unused-vars/#how-to-use